{
    "type": "bundle",
    "id": "bundle--5a0b202b-9734-4f79-a45d-6e2a8d0f8385",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "id": "relationship--7431f8d2-32df-4b26-8b17-24f10f861fde",
            "type": "relationship",
            "created": "2021-02-09T14:35:39.785Z",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "external_references": [
                {
                    "source_name": "ESET Bad Rabbit",
                    "url": "https://www.welivesecurity.com/2017/10/24/bad-rabbit-not-petya-back/",
                    "description": "M.L\u00e9veille, M-E.. (2017, October 24). Bad Rabbit: Not\u2011Petya is back with improved ransomware. Retrieved January 28, 2021."
                }
            ],
            "modified": "2021-05-04T19:28:12.894Z",
            "description": "[Bad Rabbit](https://attack.mitre.org/software/S0606) enumerates open SMB shares on internal victim networks.(Citation: ESET Bad Rabbit)",
            "relationship_type": "uses",
            "source_ref": "malware--2eaa5319-5e1e-4dd7-bbc4-566fced3964a",
            "target_ref": "attack-pattern--3489cfc5-640f-4bb3-a103-9137b97de79f",
            "x_mitre_version": "1.0",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}